src/models/Variable.js

```diff
@@ -0,0 +1,55 @@
+import FileInformation from './FileInformation';
+
+class Variable extends FileInformation {
+  /**
+   * Default constructor.
+   * @param {object} [props] - Object that contains all properties to set.
+   * @param {string} [props.name] - The name of this Variable.
+   * @param {string} [props.type] - The type of this Variable.
+   * @param {(string | boolean | number | Array)} [props.value] - The value of this Variable.
+   * @param {string} [props.category] - The variable block type.
```

Comment:
`The variable block type.`: Is not understandable, please rewrite it.

Reply:
Done.